rclcpp_examples/test/test_executables_example.py.in

```diff
@@ -0,0 +1,61 @@
+from importlib.machinery import SourceFileLoader
+import os
+from launch import LaunchDescriptor
+from launch.exit_handler import ignore_exit_handler
+from launch.launcher import DefaultLauncher
+from launch_testing import InMemoryHandler
+
+
+def setup():
+    os.environ['OSPL_VERBOSITY'] = '8'  # 8 = OS_NONE
+
+
+def test_executable():
+    output_handlers = []
+
+    ld = LaunchDescriptor()
+
+    i = 0
+    for exe, output_file in zip('@RCLCPP_EXAMPLES_EXECUTABLE@'.split(';'),
+                                '@RCLCPP_EXAMPLES_EXPECTED_OUTPUT@'.split(';')):
```

Comment:
The generated code might get pretty long. The following would be easier to read.

    executables = '@RCLCPP_EXAMPLES_EXECUTABLE@'.split(';')
    output_files = '@RCLCPP_EXAMPLES_EXPECTED_OUTPUT@'.split(';'))
    for exe, output_file in zip(executables, output_files):                            :